js/npm/arangojs/Database.hx

```diff
@@ -14,7 +14,7 @@ typedef DatabaseOptions = {
 	?promise : Dynamic
 }
 
-typedef Aqb = String;
+typedef Aqb = Either<String, { function toAQL() : String; }>;
```

Comment:
I just used the same `js.support.Either` abstract than for the rest of the extern. It also works with EitherType.